### ISSUE TITLE
9130 (product added to order) - Improve text?

### ISSUE THREAD

**Issue by DannyWPWS** [wpwhitesecurity/wsal-woocommerce#139]
It could be me but the main event text does not quite read correctly to me, should we tweak this?

![image](https://user-images.githubusercontent.com/59394454/161076396-a31e253c-b3c3-4c1a-ae97-b54453d015fb.png)

**Comment by robert81** [comment]
I agree 100%, however, I do not have an answer. What do you recommend @DannyWPWS ??
